templates/tfengine/components/cicd/configs/run.sh

```diff
@@ -39,11 +39,14 @@ do
 done
 
 ROOT=$(realpath "${ROOT}")
-IFS=', ' read -r -a args <<< "${ACTION}"
 
-for d in "${DIRS[@]}"
+for mod in "${MODULES[@]}"
 do
-    cd "${ROOT}"/"${d}"
+    cd "${ROOT}"/"${mod}"
     terraform init
-    terraform "${args[@]}"
+    for action in "${ACTIONS[@]}"
+    do
+      IFS=' ' read -r -a args <<< "${action}"
```

Comment:
Please add a comment, it's not obvious to me what this does or why it's needed.

Reply:
Done. Note the main motivation behind these changes is passing shell check

Reply:
Do we have shellcheck in CI?

Reply:
Not at the moment (I think most of our current scripts would fail it too). I think it's more important for the cloud build scripts to pass shell check as customers would use them. Filed https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/460 to track